docs/beginning-pyscript.md

```diff
@@ -226,7 +226,7 @@ and click the "run" button. Assuming you've copied the code properly, you
 should have a fine old time using "Polyglot 🦜" to translate English to
 Pirate-ish.
 
-Alternatively, [click here to see a working example of this app](https://ntoll.pyscriptapps.com/piratical/latest/).
+Alternatively, [click here to see a working example of this app](https://ntoll.pyscriptapps.com/piratical/v2).
```

Comment:
It be best to just leave the default version of the app...

Also, we should move all examples to the examples account and not have anything link to personal accounts anymore